.travis.yml

```diff
@@ -27,9 +27,14 @@ jobs:
       name: Terraform Lint/Format Verification
       install:
         - make terraform/install
-        - make terraform-docs/install
       script:
         - make terraform/lint
+    - stage: lint
+      name: Terraform Docs Format Verification
+      install:
+        - make terraform/install
```

Comment:
I don't believe terraform needs to be installed for terraform-docs

Reply:
It fails without terraform installed
> terraform not available